troubleshooting.md

```diff
@@ -210,19 +210,13 @@ cannot find newuidmap: exec: "newuidmap": executable file not found in $PATH
 
 Install a version of shadow-utils that includes these executables.  Note RHEL7 and Centos 7 will not have support for this until RHEL7.7 is released.
 
-### 10) podman fails to run in user namespace because /etc/subuid is not properly populated.
+### 10) rootless setup user: invalid argument
 
-Rootless podman requires the user running it to have a range of UIDs listed in /etc/subuid and /etc/subgid.
+Rootless podman requires the user running it to have enough subUIDs and subGIDs for running the container.
 
 #### Symptom
 
-If you are running podman or buildah as a user, you get an error complaining about
-a missing subuid ranges in /etc/subuid.
-
-```
-podman run -ti fedora sh
-No subuid ranges found for user "johndoe" in /etc/subuid
-```
+An user, either via --user or through the default configured for the image, is not mapped inside the namespace.
```

Comment:
We should mention the exact error message here

Reply:
fixed